infra/resources/prod/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "<= 3.97.1"
+    }
+
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "terraform-state-rg"
+    storage_account_name = "tfappprodio"
```

Comment:
```suggestion
    storage_account_name = "tfinfprodio"
```

Reply:
Do we have a rationale? Afaik we are moving all the state files into `tfappprodio` cc @Krusty93

Reply:
`inf` is for infrastructure where we can be a possible secrets in terraform state file